packages/sw-background-sync-queue/src/lib/requestManager.js

```diff
@@ -0,0 +1,136 @@
+import {initializationDefaults} from './constants';
+import queue from './queue.js';
+import responseManager from './responseManager';
+let globalConfig = initializationDefaults;
```

Comment:
See other comments about preferring configuration of class instances via variables passed to the constructor and stored in member variables, rather than static configuration used outside of the class structure.